pkg/objectstate/rte/rte.go

```diff
@@ -407,3 +416,20 @@ func getObject(ctx context.Context, cli client.Client, key client.ObjectKey, obj
 	*err = cli.Get(ctx, key, obj)
 	return *err == nil
 }
+
+func findConfigMapState(name, namespace string, configMaps map[string]configMapManifest) *corev1.ConfigMap {
+	var desiredConfigMap *corev1.ConfigMap
+	existingConfigMap, ok := configMaps[name]
+	errFormat := fmt.Sprintf("failed to find config map %s/%s", namespace, name)
```

Comment:
it seems we can log this in few flows, but all the flows should be mutually exclusive. So I'd avoid this and inline the formatting in the logging sites.